src/modalities/dataloader/large_file_lines_reader.py

```diff
@@ -41,6 +55,13 @@ def __init__(self, raw_data_path: Path, index_path: Optional[Path] = None):
         with self.index_path.open("rb") as f:
             self.index = pickle.load(f)
 
+        self.raw_data_fd = self.raw_data_path.open("rb")
+        self.mmapped_data_file = mmap.mmap(self.raw_data_fd.fileno(), 0, access=mmap.ACCESS_READ)
+
+    def close(self):
```

Comment:
The close-function is never called in our codebase. What is the purpose of adding it then? Also, if we keep this function, we should add an open-function as well where we open the appropriate files again

Reply:
I created the issue  #271  for this.